src/standalone/Vegetation/Canopy.jl

```diff
@@ -920,101 +920,12 @@ function ClimaLand.make_update_aux(
         # the current time, as they depend on prescribed fields.
         set_canopy_prescribed_field!(canopy.hydraulics, p, t)
 
-        # Shortcut names
-        An = p.canopy.photosynthesis.An
-        ψ = p.canopy.hydraulics.ψ
-        ϑ_l = Y.canopy.hydraulics.ϑ_l
-        fa = p.canopy.hydraulics.fa
-        par_d = p.canopy.radiative_transfer.par_d
-        nir_d = p.canopy.radiative_transfer.nir_d
-        cosθs = p.drivers.cosθs
-        area_index = p.canopy.hydraulics.area_index
-        LAI = area_index.leaf
-        SAI = area_index.stem
-
-        bc = canopy.boundary_conditions
-
-        # update radiative transfer
-        (; G_Function, Ω, λ_γ_PAR) = canopy.radiative_transfer.parameters
-        @. p.canopy.radiative_transfer.ϵ =
-            canopy.radiative_transfer.parameters.ϵ_canopy *
-            (1 - exp(-(LAI + SAI))) #from CLM 5.0, Tech note 4.20
-        RT = canopy.radiative_transfer
-        compute_PAR!(par_d, RT, bc.radiation, p, t)
-        compute_NIR!(nir_d, RT, bc.radiation, p, t)
-
-        compute_fractional_absorbances!(
-            p,
-            RT,
-            LAI,
-            ground_albedo_PAR(
-                Val(bc.prognostic_land_components),
-                bc.ground,
-                Y,
-                p,
-                t,
-            ),
-            ground_albedo_NIR(
-                Val(bc.prognostic_land_components),
-                bc.ground,
-                Y,
-                p,
-                t,
-            ),
-        )
+        # Update p.canopy.radiative_transfer.par, .nir, .ϵ, .par_d, .nir_d
+        update_radiative_transfer!(p, Y, t, canopy.radiative_transfer, canopy)
```

Comment:
I dont think we need `t` here, but removing it would change more files. I can do this if we want! so the arguments are the same for all.